tests/config/config-list-variables.spec.ts

```diff
@@ -0,0 +1,149 @@
+    })
+
+    it("Should export fixed variables for json response", async () => {
+        await new ConfigCommand().listVariables(true, ["key-1:1.0.0", "key-2:1.0.0"], null);
+
+        expect(testTransport.logMessages.length).toBe(1);
+        expect(testTransport.logMessages[0].message).toContain(FileService.fileDownloadedMessage);
+
+        const expectedFileName = testTransport.logMessages[0].message.split(FileService.fileDownloadedMessage)[1];
+        expect(mockWriteFileSync).toHaveBeenCalledWith(path.resolve(process.cwd(), expectedFileName), JSON.stringify(fixedVariableManifests), {encoding: "utf-8"});
+
+        const variableExportRequest = parse(mockedPostRequestBodyByUrl.get("https://myTeam.celonis.cloud/package-manager/api/core/packages/export/batch/variables-with-assignments"));
+        expect(variableExportRequest).toEqual(packageKeyAndVersionPairs);
+    })
+
+    it("Should list fixed variables for non-json response and keysByVersion file mapping", async () => {
+        (fs.existsSync as jest.Mock).mockReturnValue(true);
+        (fs.readFileSync as jest.Mock).mockReturnValue(JSON.stringify(packageKeyAndVersionPairs));
+
+        await new ConfigCommand().listVariables(false, [], "key_version_mapping.json");
+
+        expect(testTransport.logMessages.length).toBe(2);
+        expect(testTransport.logMessages[0].message).toContain(JSON.stringify(fixedVariableManifests[0]));
+        expect(testTransport.logMessages[1].message).toContain(JSON.stringify(fixedVariableManifests[1]));
+
+        const variableExportRequest = parse(mockedPostRequestBodyByUrl.get("https://myTeam.celonis.cloud/package-manager/api/core/packages/export/batch/variables-with-assignments"));
+        expect(variableExportRequest).toEqual(packageKeyAndVersionPairs);
+    })
+
+    it("Should export fixed variables for json response and keysByVersion file mapping", async () => {
+        (fs.existsSync as jest.Mock).mockReturnValue(true);
+        (fs.readFileSync as jest.Mock).mockReturnValue(JSON.stringify(packageKeyAndVersionPairs));
+
+        await new ConfigCommand().listVariables(true, [], "key_version_mapping.json");
+
+        expect(testTransport.logMessages.length).toBe(1);
+        expect(testTransport.logMessages[0].message).toContain(FileService.fileDownloadedMessage);
+
+        const expectedFileName = testTransport.logMessages[0].message.split(FileService.fileDownloadedMessage)[1];
+        expect(mockWriteFileSync).toHaveBeenCalledWith(path.resolve(process.cwd(), expectedFileName), JSON.stringify(fixedVariableManifests), {encoding: "utf-8"});
+
+        const variableExportRequest = parse(mockedPostRequestBodyByUrl.get("https://myTeam.celonis.cloud/package-manager/api/core/packages/export/batch/variables-with-assignments"));
+        expect(variableExportRequest).toEqual(packageKeyAndVersionPairs);
+    })
+
+    it("Should throw error if no mapping or file path is provided", async () => {
```

Comment:
I think the test name should be `Should throw error if no mapping and no file path is provided`